src/Modules/DynamicLoading/DynamicLoading/Center.cs

```diff
@@ -0,0 +1,155 @@
+                Process proc = new Process();
+                proc.StartInfo.FileName = cmd;
+                proc.StartInfo.Arguments = arguments;
+                proc.StartInfo.UseShellExecute = false;
+                proc.StartInfo.RedirectStandardOutput = true;
+                proc.Start();
+                Console.WriteLine(proc.StandardOutput.ReadToEnd());
+                proc.WaitForExit();
+            }
+        }
+
+        public static void CreateCSProj()
+        {
+            var path = System.IO.Path.Combine(
+                        CurrentVersion.TslBuildDir,
+                        $"{Namespace}.csproj");
+            using (var s = System.IO.File.CreateText(path))
+            {
+                CSProj.TemplateLine.ForEach(s.WriteLine);
+            }
+        }
+        public static bool CodeGenFn()
+        {
+            System.IO.Directory.GetFiles(CurrentVersion.TslSrcDir, "*.tsl").ToList().ForEach(Console.WriteLine);
+            return Cmd.TSLCodeGenCmd(
+                string.Join(" ", System.IO.Directory.GetFiles(CurrentVersion.TslSrcDir, "*.tsl")) +
+                $" -offset {CurrentVersion.CellTypeOffset} -n {Namespace} -o {CurrentVersion.TslBuildDir}");
+        }
+
+        public static bool BuildFn() =>
+            Cmd.DotNetBuildCmd($"build {CurrentVersion.TslBuildDir} -o {CurrentVersion.AsmLoadDir}");
+
+        public static Assembly LoadFn() {
+            Console.WriteLine("Loading " + System.IO.Path.Combine(CurrentVersion.AsmLoadDir, $"{Namespace}.dll"));
+            return Assembly.LoadFrom(System.IO.Path.Combine(CurrentVersion.AsmLoadDir, $"{Namespace}.dll"));
+        }
+      
+        public static void LoadFrom(
+                    string tslsrcDir, 
+                    string tslBuildDir = null, 
+                    string asmLoadDir = null,
+                    string versionName = null)
+        {
+
+            CurrentVersion = new AsmVersion(
+                _currentCellTypeOffset,
+                versionName ?? DateTime.Now.ToString(),
+                tslsrcDir, tslBuildDir?? tslsrcDir, asmLoadDir??tslBuildDir??tslsrcDir);
+
+            Console.WriteLine("tslsrcDir: " + tslsrcDir +
+                              "TslBuildDir : " + CurrentVersion.TslBuildDir + 
+                              ", AsmLoadDir : " + CurrentVersion.AsmLoadDir);
+
+            try
+            {
+                CreateCSProj();
+            }
+            catch (Exception e)
+            {
+                throw e;
+            }
+            if (!CodeGenFn())
+            {
+                throw new TSLCodeGenError();
+            }
+
+            if (!BuildFn())
+            {
+                throw new TSLBuildError();
+            }
+
+            try
+            {
+                var asm = LoadFn();
+                asm.GetTypes().ToList().ForEach(Console.WriteLine);
+                _asmSlides.Add(asm);
+
+                /// TODO: get cell_type count by `asm`.
```

Comment:
use AssemblyUtility.GetAllClassInstances<IStorageSchema> to extract the schema from the generated assembly.